### PR TITLE
Nonoise

### PR DESCRIPTION
**Pull request by esheldon**
removing noise images and old sim codes

### CODE REVIEW

File: descwl_shear_sims/wcs/dmwcs.py [hunk 1]
```diff
@@ -59,9 +59,11 @@ def make_dm_wcs(galsim_wcs):
     return stack_wcs
 
 
-def make_coadd_dm_wcs(coadd_origin):
+# def make_coadd_dm_wcs(coadd_origin):
```
Comment (beckermr) [inline descwl_shear_sims/wcs/dmwcs.py:62]: remove comment here

File: setup.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 setup(
     name="descwl_shear_sims",
-    version="0.3.0",
+    version="0.3.1",
```
Comment (beckermr) [inline setup.py:5]: We should read this out of the package. I can do this in another PR.

**Review by beckermr — CHANGES_REQUESTED** [review]
Some small things and then we can merge!

File: setup.py (hunk 2 above)
Comment (esheldon) [reply setup.py:5]: I'll do it

File: descwl_shear_sims/wcs/dmwcs.py (hunk 1 above)
Comment (esheldon) [reply descwl_shear_sims/wcs/dmwcs.py:62]: ok

**Comment by esheldon** [comment]
yes!

**Review by beckermr — APPROVED** [review]

**Pull request merged**